src/openrct2/world/map_helpers.c

```diff
@@ -372,29 +375,31 @@ sint32 tile_smooth(sint32 x, sint32 y)
     sint8 thresholdSE = clamp(neighbourHeightOffset.E, 0, 1) + clamp(neighbourHeightOffset.SE, 0, 1) + clamp(neighbourHeightOffset.S, 0, 1);
     sint8 thresholdSW = clamp(neighbourHeightOffset.S, 0, 1) + clamp(neighbourHeightOffset.SW, 0, 1) + clamp(neighbourHeightOffset.W, 0, 1);
 
-    uint8 slope = 0;
-    slope |= (thresholdNW >= 1) ? (1 << 1) : 0;
-    slope |= (thresholdNE >= 1) ? (1 << 2) : 0;
-    slope |= (thresholdSE >= 1) ? (1 << 3) : 0;
-    slope |= (thresholdSW >= 1) ? (1 << 0) : 0;
-
-    // Set diagonal when three corners have been raised, and the middle one can be raised one more
-    if ((slope == 0b0111 && neighbourHeightOffset.NW >= 4) || (slope == 0b1011 && neighbourHeightOffset.SW >= 4) ||
-        (slope == 0b1101 && neighbourHeightOffset.SE >= 4) || (slope == 0b1110 && neighbourHeightOffset.NE >= 4))
+    uint8 slope = TILE_ELEMENT_SLOPE_FLAT;
+    slope |= (thresholdNW >= 1) ? SLOPE_NW_THRESHOLD_FLAGS : 0;
+    slope |= (thresholdNE >= 1) ? SLOPE_NE_THRESHOLD_FLAGS : 0;
+    slope |= (thresholdSE >= 1) ? SLOPE_SE_THRESHOLD_FLAGS : 0;
+    slope |= (thresholdSW >= 1) ? SLOPE_SW_THRESHOLD_FLAGS : 0;
+
+    // Set diagonal when three corners (one corner down) have been raised, and the middle one can be raised one more
+    if (slope == TILE_ELEMENT_SLOPE_NW_CORNER_UP && neighbourHeightOffset.NW >= 4 ||
```

Comment:
You removed too many parentheses. Travis fails because of this.

This line needs to read:
```C
if ((slope == TILE_ELEMENT_SLOPE_NW_CORNER_UP && neighbourHeightOffset.NW >= 4) ||
```
Please change this line and the other three accordingly.